optimizely/project_config.py

```diff
@@ -68,6 +70,7 @@ def __init__(self, datafile, logger, error_handler):
 
         # Utility maps for quick lookup
         self.group_id_map = self._generate_key_map(self.groups, 'id', entities.Group)
+        self.experimentID_map = self._generate_key_map(self.mapexperiment, 'id', entities.Experiment)
```

Comment:
We shouldn't mix camel cases with snake_case

Reply:
Why experiment's can't be passed? what's the need of mapexperiment?